core-aam/index.html

```diff
@@ -5711,6 +5711,62 @@ <h4>Not Mapped</h4>
               >. Its absence not only indicates that the accessible object is not grabbed, but further defines it as not grab-able. These cases are marked as "Not mapped" without an asterisk.
             </p>
           </section>
+          <h4 id="ariaActions"><code>aria-actions</code></h4>
+          <table aria-labelledby="ariaActions">
+            <tbody>
+              <tr>
+                <th>ARIA Specification</th>
+                <td>
+                  <a class="property-reference" href="#aria-actions"><code>aria-actions</code></a>
+                </td>
+              </tr>
+              <tr>
+                <th>MSAA + IAccessible2</th>
+                <td>
+                  <span class="property">
+                    Interface: <code>IAccessibleAction</code>, adding one action for each accessible node referenced by `aria-actions` that is exposed in the accessibility tree. The interface methods
+                    are mapped as follows:
+                    <ul>
+                      <li>`name`: return the DOM id of the corresponding target node.</li>
+                      <li>`localizedName`: return the corresponding target node's accessible name.</li>
+                      <li>`doAction`: behave as if `doAction(0)` were called on the corresponding target accessible node.</li>
+                    </ul> </span
+                  ><br />
+                  <span class="property">Object Attribute: <code>aria-actions: ""</code></span>
```

Comment:
I suggest has-actions: true
- In the future, the actions could come from HTML
- We don't typically name object attributes using "aria" in the name because at that point as the semantics should be technology-agnostic where possible
- The pattern typically has a value, often "true".

@jcsteh WDYT?

Reply:
makes sense to me, FWIW. But yeah, I think @jcsteh's sign-off is more important :D